amundsen_application/static/js/components/SearchPage/ResourceSelector/tests/index.spec.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { mapStateToProps, ResourceSelector, ResourceSelectorProps} from '../';
+import { TABLE_RESOURCE_TITLE, USER_RESOURCE_TITLE } from 'components/SearchPage/constants';
+
+import AppConfig from 'config/config';
+import globalState from 'fixtures/globalState';
+import { ResourceType } from 'interfaces/Resources';
+
+
+describe('ResourceSelector', () => {
+  const setup = (propOverrides?: Partial<ResourceSelectorProps>) => {
+    const props = {
+      selectedTab: ResourceType.table,
+      tables: globalState.search.tables,
+      users: globalState.search.users,
+      dashboards: globalState.search.dashboards,
+      onChange: jest.fn(),
+      ...propOverrides
+    };
+    const wrapper = shallow<ResourceSelector>(<ResourceSelector {...props} />);
+    return { props, wrapper };
+  };
+
+  describe('onChange', () => {
+    // TODO - placeholder as I expect the `onChange` to disappear once we use redux for search state actions
+  });
+
+  describe('renderRadioOption', () => {
+    // TODO - placeholder as I expect the radio buttons logic to change
```

Comment:
@danwom what was the context of this again? I will clean up before merge.

Reply:
Right now the change selected resource handler is passed into the `ResourceSelector` since that state was previously managed in `SearchPage`. Now that we've updated the search redux logic, the `setResource` action can be passed directly into the `ResourceSelector` instead of being passed through from the parent components.